src/stories/setup.js

```diff
@@ -37,14 +37,12 @@ library.add(
   faSearch,
   faBars,
   faTimes,
+  faCheck,
 )
+library.add(faExclamation)
```

Comment:
why is this separated?

Reply:
Lapsus, forgot to remove it. 
On a second note, some of the font awesome icons didnt work for me. (question,exclamation...)

Reply:
If others work, so should these 2. Maby you had a typo or something

Reply:
Nope, it was literally the same as everything else. I tried all the possible combinations (fas, far, camelcase, snakecase, kebabcase)
If you have the time you can try the question icon and tell me if it works for you